docs/run-keep-ecdsa.adoc

```diff
@@ -348,7 +359,23 @@ Trust math, not hardware.
 -----------------------------------------------------------------------------------------------
 ```
 
-**Bonus**: If you want to share your LibP2P address with others you can get it from the startup log.  When sharing remember to substitute the `/ipv4/` address with the public facing IP of your client if you're running on a private machine, or replace the entire `/ipv4/` segment with a DNS entry if you're using a hostname.
+If you want to share your LibP2P address with others you can get it
+from the startup log. This can be helpful for debugging issues where a peer ID
+is needed. Additionally, if you're running multiple nodes, you may want to add
+your own nodes to the bootstrap list configured in `LibP2P.Peers`.
```

Comment:
Maybe worth saying explicitly they need to be added next to the network's official bootstrap nodes?

Want to avoid someone setting their nodes as the only bootstraps.